@NeOMakinG You are a magician ! When you change CSS and HTML you break the SQL engine feature :trollface: 

```
Run php install-dev/index_cli.php --language=en --country=fr --domain=localhost --db_server=127.0.0.1 --db_password=password --db_name=prestashop --db_create=1 --name=prestashop.unit.test --email=demo@prestashop.com --***
PrestaShopException: Cannot select any valid SQL engine. in /home/runner/work/PrestaShop/PrestaShop/classes/db/Db.php:302
Stack trace:
#0 /home/runner/work/PrestaShop/PrestaShop/src/PrestaShopBundle/Install/Database.php(123): DbCore::getClass()
#1 /home/runner/work/PrestaShop/PrestaShop/install-dev/controllers/console/process.php(115): PrestaShopBundle\Install\Database->createDatabase('127.0.0.1', 'prestashop', 'root', 'password')
#2 /home/runner/work/PrestaShop/PrestaShop/install-dev/classes/controllerConsole.php(117): InstallControllerConsoleProcess->process()
#3 /home/runner/work/PrestaShop/PrestaShop/install-dev/index_cli.php(64): InstallControllerConsole::execute(11, Array)
#4 {main}
Error: Process completed with exit code 1.
```

Re-running CI

--- a/admin-dev/themes/new-theme/scss/pages/_payment.scss
+++ b/admin-dev/themes/new-theme/scss/pages/_payment.scss
@@ -39,5 +39,59 @@
         }
       }
     }
+
+    .tab-content {
+      @include media-breakpoint-down(sm) {
+        padding: 0 0.5rem;
+      }
+    }
+
+    .tab-pane {
+      @include media-breakpoint-down(sm) {
+        .table {
+          tr {
+            position: relative;
+            display: flex;
+            flex-wrap: wrap;
+            width: 100%;
+            min-width: 12rem;
+            padding: 1rem 0;
+            padding-left: 4.375rem;
+
+            &:not(:last-child) {
+              border-bottom: 1px solid $gray-light;
+            }
+
+            td {
+              padding: 0;
+              border: none;
+
+              &:first-child {
+                position: absolute;
+                left: 0;
+              }
+
+              &:last-child {
+                .btn-group {
+                  margin-top: 0;
+                }
+              }
+            }
+
+            .module_name {
+              .text-muted {
+                display: block;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .alert-block {
+    @include media-breakpoint-down(sm) {
+      padding: 0;
+    }
   }
 }

--- a/src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/payment_methods.html.twig
+++ b/src/PrestaShopBundle/Resources/views/Admin/Improve/Payment/PaymentMethods/payment_methods.html.twig
@@ -29,7 +29,7 @@
   <div class="container">
 
     {% if paymentModules|length < 2 %}
-      <div class="card-block row">
+      <div class="card-block row alert-block">
         <div class="card-text">
           <div class="row">
             <div class="col-sm">

